lib/src/participant/local.dart

```diff
@@ -158,6 +159,13 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
     logger.fine(
         'Compute encodings with resolution: ${dimensions}, options: ${publishOptions}');
 
+    if (isSVCCodec(publishOptions.videoCodec)) {
+      // set scalabilityMode to 'L3T3_KEY' by default
+      publishOptions = publishOptions.copyWith(
+        scalabilityMode: 'L3T3_KEY',
```

Comment:
this will always override `scalabilityMode`, right? 
I think we would only want to do that in case `scalabilityMode` isn't set within `publishOptions` already